pkg/postgres/walker/walker.go

```diff
@@ -114,19 +180,27 @@ func getPostgresOptions(tag string, topLevel bool, ignorePK, ignoreUnique bool)
 		case field == "unique":
 			opts.Unique = !ignoreUnique
 		case strings.HasPrefix(field, "fk"):
-			// If we already have a field acting as foreign key use it as is rather than adding a redundant field using parentify.
+			if ignoreFKs {
```

Comment:
Yeah, in the children. I did this for the alert->image case, since alert embed a `ContainerImage` which has an `fk` to Image -- _maybe_ it makes sense to have alerts and images linked to enable those searches, but I did not want to change behaviour in this PR.

Reply:
makes sense